packages/integration-tests/src/tests/experience/first-screen.test.ts

```diff
@@ -0,0 +1,209 @@
+import { ConnectorType } from '@logto/connector-kit';
+import { SignInIdentifier } from '@logto/schemas';
+
+import { updateSignInExperience } from '#src/api/sign-in-experience.js';
+import { demoAppUrl } from '#src/constants.js';
+import {
+  clearConnectorsByTypes,
+  setEmailConnector,
+  setSmsConnector,
+} from '#src/helpers/connector.js';
+import ExpectExperience from '#src/ui-helpers/expect-experience.js';
+import { devFeatureTest } from '#src/utils.js';
+
+const { describe, it } = devFeatureTest;
+
+describe('first screen', () => {
+  beforeAll(async () => {
+    await clearConnectorsByTypes([ConnectorType.Social, ConnectorType.Email, ConnectorType.Sms]);
+    await setEmailConnector();
+    await setSmsConnector();
+  });
+
+  describe('sign-in page', () => {
+    it('should be landed on sign-in page directly', async () => {
+      const experience = new ExpectExperience(await browser.newPage());
+      const url = new URL(demoAppUrl);
+      url.searchParams.set('first_screen', 'sign_in');
+      await experience.page.goto(url.href);
+      await experience.page.waitForNetworkIdle();
+      experience.toBeAt('sign-in');
+      await experience.page.close();
+    });
+  });
```

Comment:
Should we also add a test case to cover the default behavior if no "first_screen" param is specified? I think we should fallback to sign-in, right?

Reply:
The flow you mentioned has already been covered in other tests.